DQM/L1TMonitorClient/data/L1TStage2BMTFQualityTests.xml

```diff
@@ -28,4 +28,20 @@
       <TestName activate="true">BMTF_WedgeBXNoisyWedge</TestName>
   </LINK>
 
+  <!-- Bad Zero Suppression Rates -->
+
+  <QTEST name="zeroSupp_MismatchRatioMax0">
```

Comment:
Question to @jfernan2 @dmitrijus : Does a quality test name need to be unique? The same Q test name already exists in a different xml: https://github.com/cms-sw/cmssw/blob/master/DQM/L1TMonitorClient/data/L1TStage2uGMTQualityTests.xml#L385

Reply:
Pinging @rovere

Reply:
hi all,
IRCC you can use the same name as long as the histograms tested are in different folders since the results of the QT will be an "XML-formatted" string saved in that folder.
I suggest having a closer look at the code, though.

Reply:
Hi,
thanks for the answer. I had a look at the code and that seems right. Each QualityTester gets its own xml and does not need to know about others.
@jfernan2 @dmitrijus can you please approve this PR and also the one to the master?
@panoskatsoulis did you already request this to be deployed at P5?